### DISCUSSION TITLE
Issues with NGio and ModMenu

### DISCUSSION THREAD

**Discussion by FNFFAN737** [FunkinCrew/Funkin#2104] [Compiling help]
Compiling the game normally gives me these errors, mostly in regards to NGio.hx but some in regards to ModMenu.hx . Not sure what I should do here. For context, this is the most recent version of the Funkin github, now with the Week 7 code.

![DB7E8843-40C8-40A3-823C-8B28B1761071](https://user-images.githubusercontent.com/96362136/215909436-0b28251d-bbea-4df1-8bd0-ee7045b1283a.jpeg)

**Comment by CrusherNotDrip** [comment]
https://github.com/FunkinCrew/Funkin/issues/2077
